pkg/cmd/repo/fork/fork.go

```diff
@@ -195,6 +195,13 @@ func forkRun(opts *ForkOptions) error {
 		if err != nil {
 			return err
 		}
+
+		if remote, err := remotes.FindByRepo(repoToFork.RepoOwner(), repoToFork.RepoName()); err == nil {
+			if remote.FetchURL.Scheme == "https" || remote.FetchURL.Scheme == "ssh" {
```

Comment:
This might be super edge-case-y, but `remote.FetchURL` might be nil if there is a non-nil `PushURL`. Access to either should be guarded by a non-nil check.

Additionally, since the ultimate role of the new remote is to push changes to it, we might be more interested to prioritize the protocol of PushURL (if it exists) over that of the FetchURL, in case someone is using different protocols for reads and for pushes.

Reply:
makes sense.